src/styles/main.scss

```diff
@@ -0,0 +1,65 @@
+$daySize: 100px;
+$countDays: 31;
+$gap: 1px;
+$calendarWidth: $daySize * 7 + 10;
```

Comment:
The calculation for `$calendarWidth` seems incorrect. The formula `$daySize * 7 + 10` does not account for the gaps between the days. You should include the gaps in the calculation to ensure the correct width.